### PR TITLE
Add config and code to build Docker image for operations playground

### PR DESCRIPTION
**Pull request by fhueske**
This PR builds on #1 by @knaufk and adds the Java code and Docker configuration files to build a custom Docker image for the operations playground.

Right now, we expect users to build the image themselves by running `docker-compose build` once. Later we plan to integrate with Apache's infrastructure to automatically build and publish Docker images to Docker Hub.

The PR is based on Flink 1.8.1 because it assumes an existing Flink Docker base image (https://hub.docker.com/_/flink).

The code of the example job was copied from the [Flink main repository](https://github.com/apache/flink/tree/master/flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/windowing/clickeventcount) and adapted to Flink 1.8.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Apache Flink Playgrounds
+
+This repository provides playgrounds to quickly and easily explore [Apache Flink](https://flink.apache.org)'s features.
+
+The playgrounds based on [docker-compose](https://docs.docker.com/compose/) environments.
```
Comment (rmetzger) [inline README.md:5]:
```suggestion
The playgrounds are based on [docker-compose](https://docs.docker.com/compose/) environments.
```

File: operations-playground/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+To stop the playground, run the following command
+
+```bash
+docker-compose down
+```
+
+## Further instructions
+
+The playground setup and more detailed instructions are presented in the
+["Getting Started" guide](https://ci.apache.org/projects/flink/flink-docs-release-1.8/getting-started/docker-playgrounds/flink-cluster-playground.html) of Flink's documentation.
```
Comment (rmetzger) [inline operations-playground/README.md:50]:
```suggestion
["Getting Started" guide](https://ci.apache.org/projects/flink/flink-docs-release-1.8/getting-started/docker-playgrounds/flink_cluster_playground.html) of Flink's documentation.
```

**Comment by fhueske** [comment]
Thanks for the review @rmetzger!
I'll merge the playground for Flink 1.8 and will update it to Flink 1.9 once it is released

**Comment by fhueske** [comment]
Merged

**Pull request closed without merging**